src/parser/jobs/mnk/modules/PerfectBalance.tsx

```diff
@@ -0,0 +1,45 @@
+import {t, Trans} from '@lingui/macro'
+import React from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import {Action} from 'data/ACTIONS/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {Status} from 'data/STATUSES/STATUSES'
+import {BuffWindowModule} from 'parser/core/modules/BuffWindow'
+import {SEVERITY} from 'parser/core/modules/Suggestions'
+
+export default class PerfectBalance extends BuffWindowModule {
+	static handle = 'pb'
+	static title = t('mnk.pb.title')`Perfect Balance Windows`
+
+	buffAction: Action = ACTIONS.PERFECT_BALANCE
+	buffStatus: Status = STATUSES.PERFECT_BALANCE
+
+	expectedGCDs = {
+		expectedPerWindow: 6,
```

Comment:
This kinda depends on what Fist the player is in though, and if they're in stack recovery. I think it's ok for now but once the MNK gauge is done will wanna rethink this I suppose. AFAIK BuffWindowModule has some extra methods for customising this logic so should be ok. More of a note than a thing that needs to change though.

Reply:
It does, but my discussion with Tiff implied to me that maximum Wind uptime is desirable.

What we could do is a set a baseline of 5, since that is always possible even in recovery, and then let 6th be a bonus that has to be checked by a human?

Reply:
I think that's a good idea. There's no reason to get less than 5, it's just deciding if 5 is acceptable or not depending on stance/stacks. We already have checks for maximum wind but dunno if we should use PB expected GCDs to reinforce it.

BuffWindowModule exposes `changeExpectedGCDsClassLogic(buffWindow: BuffWindowState): number` which can check the current Fists or GL and then return the amount to increment/decrement the expectedPerWindow value. Not sure if better to use stance or stacks as that metric though.

Reply:
I think realistically we need full stack support and lookback to support mandating 6. I'd rather not dig down that rabbit hole as of now.